lib/cors-anywhere.js

```diff
@@ -10,10 +10,10 @@ var regexp_tld = require('./regexp-top-level-domain');
 var getProxyForUrl = require('proxy-from-env').getProxyForUrl;
 var requiresPort = require('requires-port');
 
-var help_file = __dirname + '/help.txt';
 var help_text;
-function showUsage(headers, response) {
-  headers['content-type'] = 'text/plain';
+function showUsage(help_file, headers, response) {
+  var isHtml = (help_file.indexOf('.html')) != -1;
```

Comment:
`var isHtml = /\.html$/.test(help_file);`

because:

- Case-insensitive
- filename.html.txt is a text file, not a HTML file.